openedx/frontend-app-authoring
# fix: remove preview overlay from component sidebar

## Description

This PR removes the div overlay that prevents users from interacting with the component preview on the sidebar

## More Information

- https://github.com/openedx/frontend-app-authoring/issues/1189#issuecomment-2372395553

## Testing Instruction
- Checkout this PR
- Open the component sidebar and check if you can interact with the content (play a video, select text, etc..)
